pkg/runner/runner.go

```diff
@@ -32,19 +32,43 @@ type K6Runner struct {
 	Params Params
 }
 
+const K6_CLOUD = "cloud"
+const K6_RUN = "run"
+const K6_SCRIPT = "script"
+
 func (r *K6Runner) Run(execution testkube.Execution) (result testkube.ExecutionResult, err error) {
 	// check that the datadir exists
 	_, err = os.Stat(r.Params.Datadir)
 	if errors.Is(err, os.ErrNotExist) {
 		return result, err
 	}
 
-	args := []string{"run"}
+	args := []string{}
+
+	k6TestType := strings.Split(execution.TestType, "/")
+	if len(k6TestType) != 2 {
+		return result.Err(fmt.Errorf("invalid test type %s", execution.TestType)), nil
+	}
+
+	k6Subtype := k6TestType[1]
+	if k6Subtype == K6_SCRIPT || k6Subtype == K6_RUN {
+		args = append(args, K6_RUN)
+	} else if k6Subtype == K6_CLOUD {
+		args = append(args, K6_CLOUD)
+	} else {
+		return result.Err(fmt.Errorf("unsupported test type %s", execution.TestType)), nil
+	}
 
 	// convert executor env variables to k6 env variables
 	for key, value := range execution.Envs {
-		env_var := fmt.Sprintf("%s=%s", key, value)
-		args = append(args, "-e", env_var)
+		if key == "K6_CLOUD_TOKEN" {
+			// set as OS environment variable
+			os.Setenv(key, value)
+		} else {
+			// pass to k6 using -e option
+			env_var := fmt.Sprintf("%s=%s", key, value)
```

Comment:
Camel case car names in go

Reply:
Done. Renamed to comply with Go conventions.